README.md

```diff
@@ -17,48 +17,61 @@ For all the details and features of Ansible see:
 - [Ansible Best Practices](https://docs.ansible.com/ansible/latest/user_guide/playbooks_best_practices.html)
 - [Ansible Modules](https://docs.ansible.com/ansible/latest/modules/modules_by_category.html)
 
-### Development and Usage
+### Usage
 
-To deploy the specified configurations to your managed machines it is required to specify hosts in an 'inventory' file. It is often convenient to specify two of these inventory files, a 'production' and a 'staging' (or 'testing') file. These can be populated with the respective fully qualified host names of your managed machines and then specified at the command line as desired on the control machine.
+#### Info
+To deploy the specified configurations to your managed machines it is required to specify hosts in an 'inventory' file. It is often convenient to specify two of these inventory files, a 'production' and a 'staging' (or 'testing') file, if the resources for both are available. These can be populated with the respective fully qualified host names of your managed machines and then specified at the command line as desired on the control machine. There is a sample inventory at the base level of the repo, `sample.hosts`.
 
-A sample testing inventory file, named hosts.test, would look like this
-```
-[data]
-test-data.my.org
-
-[index]
-test-indexidp.my.org
-
-[idp]
-test-indexidp.my.org
-```
+The second important file that will be unqiue for each sites deployment is a variable file. There is a sample variables file that contains all available options and info in the base level of the repo, `sample.vars.yml`.
 
+#### Examples
 Ansible assumes the use of keys for ssh authentication. It provides `--ask-pass` and `-u [user]` to ssh via password authentication. For escalated privileges, if sshing as a non-root user, `--ask-become-pass` is used.
 
-A test deployment to all managed test hosts, with ssh via the root user and password authentication.
+A test deployment to all managed test hosts, with ssh via the root user and password authentication. Note the '@' is required.
 ```
-ansible-playbook -i hosts.test all.yml --ask-pass -u root --tags install
+ansible-playbook -i hosts.test --ask-pass -u root -e @myvars.test.yml install.yml
```

Comment:
Suggested change: ```ansible-playbook -i hosts.test --ask-pass -u root -e @<my.variable.file> install.yml```
where my.variable.file is the path to your variable file.

Or alternatively: ```ansible-playbook -i hosts.test --ask-pass -u root -e @sample.vars.yml install.yml```

Something more explicitly state that the -e option accepts the variable file as input.  I didn't see a reference to myvars.test.yml before it was used so it wasn't immediately clear what it meant.

Reply:
See above comment on line 30 for better suggestion.